data/hai/hai missions.txt

```diff
@@ -1932,3 +1932,73 @@ mission "Station Exploration Abandon Ship"
 			label fine
 			`	"You are being fined <fine> for abandoning your passengers without warning. You're lucky you didn't end up with a more serious punishment." After further chewing you out for endangering your passengers' safety, he walks away from you in a huff.`
 
+mission "Hai Pilgrimage"
+	name "Hai spiritual leader to <planet>"
+	description "Bring a Hai spiritual leader to <destination>. He will pay you <payment>."
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	destination
+		attributes "unfettered"
+	passengers 1
+	to offer
+		random < 30
```

Comment:
That might be a bit too likely to trigger since it offers everywhere
10% maybe?